### PR TITLE
Refactor label creation, add 'Release' PR support

### PR DESCRIPTION
**Pull request by colinta**
All labels are now in an array of hashes, hopefully makes those clearer.

Adds notion of a "Release" PR, we use them in iOS land to group together lots of PRs.

### CODE REVIEW

File: Dangerfile [hunk 1]
```diff
@@ -25,34 +25,42 @@ def remove_label_from_issue!(repo, pr_number, label)
...
-if git.lines_of_code < 50 && !issue_has_tiny_label
-  add_labels_to_issue!(repo, pr_number, ['tiny'])
-elsif git.lines_of_code > 50 && issue_has_tiny_label
-  remove_label_from_issue!(repo, pr_number, 'tiny')
+labels = [
+  {name: 'tiny', color: 'f7c6c7'},
+  {name: 'please review', color: 'fbca04'},
+  {name: 'reviewed', color: '0e8a16'},
+  {name: 'wip', color: '1d76db'},
+  {name: 'release', color: '336694'},
```
Comment (rynbyjn) [inline Dangerfile:33]: Nice, I've wanted to normalize these a bit 🤘

**Review by rynbyjn — APPROVED** [review]
LGTM 🤘

**Comment by colinta** [comment]
@rynbyjn One more refactor: https://github.com/ello/danger/pull/1/commits/616391e8ab060c75a59dffa30f3012e0e2324caf

**Comment by colinta** [comment]
@rynbyjn i.e.
```ruby
  if !issue_has_label?('reviewed') && !issue_has_label?('please review')
    add_label_to_issue!('please review')
  end
```

File: Dangerfile [hunk 2]
```diff
@@ -1,27 +1,27 @@
-repo = github.pr_json[:base][:repo][:full_name]
-pr_number = github.pr_json[:number]
+REPO = github.pr_json[:base][:repo][:full_name]
+PR_NUMBER = github.pr_json[:number]
```
Comment (rynbyjn) [inline Dangerfile:2]: Cool

**Pull request merged**